### DISCUSSION TITLE
IF Statement

### DISCUSSION THREAD

**Discussion by mrkrich** [datalust/seq-tickets#1260] [Help] [answered]
Hello All. I'm newbie for SEQ. 

I created a dashboard from query but the legend shown in graph is hard to read. 
Can I transform it to readable legend?

I try to use IF/Then/Else Statement as I can see in cheat sheet and document but I can't find where and how to use it. So, I try it like this 

`select if layer = 'null' then 'Focus Layer' else layer as TheLayer, count(*) as count from stream group by layer, time(1m) limit 100`

and I got this as error

`
The query could not be executed.
Syntax error (line 1, column 11): unexpected identifier 'layer'.
`

![image](https://user-images.githubusercontent.com/31063913/126737179-36279482-824f-484e-8ab6-eb95fb07d773.png)
![image](https://user-images.githubusercontent.com/31063913/126737219-3b002f9f-d36a-49d9-aa31-9cdb9e869cc2.png)

**Comment by nblumhardt** [comment, marked as answer]
Hi @mrkrich - your query looks fine, and runs successfully for me. Are you using an older Seq version? (The current version is 2021.2; I think `if` was added late in the 2020.x series.)

**Reply by mrkrich** [reply]
Oh! It seem like I'm using older version as you said. my SEQ version is only 5.1.3200. 

Thank you.
